src/IQFeed.CSharpApiClient/Extensions/StringExtensions.cs

```diff
@@ -41,6 +42,27 @@ public static string[] SplitFeedMessage(this string s)
             return s.Split(IQFeedDefault.ProtocolDelimiterCharacter);
         }
 
+        public static string[] SplitFeedMessage(this string s, int expectedParts)
```

Comment:
not sure here. Ill review more.

Reply:
this solves the case when the last part of the message may have commas
e.g. in http://www.iqfeed.net/dev/api/docs/SymbolLookupviaTCPIP.cfm
please check out SBN,[Search String],[RequestID]<CR><LF>
>> Description - The name of the company or a description of what the symbol represents. 
NOTE: This field may contain commas. As a result, it will ALWAYS be the last field in this message.